src/main/discord_bot.py

```diff
@@ -43,5 +39,5 @@ async def background_tasks(self):
 if __name__ == '__main__':
     config = json.loads(secrets_loader.get_secret('discord-bot'))
 
-    client = DiscordBot()
+    client = DiscordBot(status_check_config=config['status.check.config'])
```

Comment:
Is `status_check_config` necessary?
```suggestion
    client = DiscordBot()
```

Reply:
yes

Reply:
see https://github.com/MarkoDedovic/discord-bot/pull/5#discussion_r657636960